app/Http/Kernel.php

```diff
@@ -61,6 +61,7 @@ class Kernel extends HttpKernel
         'throttle' => \Illuminate\Routing\Middleware\ThrottleRequests::class,
         'verified' => \Illuminate\Auth\Middleware\EnsureEmailIsVerified::class,
         'admin' => \App\Http\Middleware\MustBeAdmin::class,
+        'verify_quiz' => \App\Http\Middleware\VerifyQuiz::class,
```

Comment:
it may sound a little more verbose... but `quiz_token_verified` makes more sense for the route middleware name.